docs/userguide/minimum-requirements.md

```diff
@@ -0,0 +1,30 @@
+# Minimum Requirements
+
+## Minimum Requirements - no proxy sidecar
+
+At a minimum, you will need the following (with example files included in the release):
+
+- Server where you will be deploying your release
+- Configuration file (examples included in the release)
+- Release archive - A versioned .zip file Stored in S3 containing:
+  - release.json - (examples included in the release) - a config for deployment including predeploy, deploy and stop commands to execute. Also declares template files.
+  - **_Optional_** Template files - mustache template files for configuration of other components such as proxy, fluentd and docker compose (based on deploy configuration).
+  - **_Optional_** Deployment Scripts
+  - **_Optional_** Binary to be deployed (the deploy command/script can source the binary)
+- **_Optional_** AWS Credentials: see the [official AWS credential page](https://aws.amazon.com/blogs/security/a-new-and-standardized-way-to-manage-credentials-in-the-aws-sdks/)
+- S3 bucket(s) with necessary permissions for r/w - release store and optional state store
+- Folders with necessary permissions for writing logs
```

Comment:
I'd write something like:

- The following working directories, writable by the user(s) who will run hx-deploy-tool.

```
/opt/releases     # where releases get unpacked for deployment
/opt/etc/deployment  # where deployment contexts get written
/opt/var/log/    # where the log file gets written
```

Note that the actual directories used can be overwritten in the configuration file.

Reply:
added with minor changes